.travis.yml

```diff
@@ -32,13 +32,16 @@ matrix:
     - python: '3.6'
       env: TOXENV="py-travis"
 
-    - python: '3.6' # Only runs checks on 3rd party for latest Python distribution
+    - python: '3.6' # To be removed when 3.7 doesn't need sudo.
```

Comment:
Is this comment intended?

Reply:
Whoops nope.

Reply:
Oh wait, yes, that's intended. Because eventually, we only want to test 1 environment for third party. Now it's doing both py 3.6 and 3.7 for third party tools.

Reply:
Yes, that should be the goal. The third party tests puts extra loads on the academic sites hosting the data/models and it takes quite some time to complete so I think it's best to minimize the tests on the third party tools build.